frontend/src/metabase/admin/people/components/NudgeToPro.tsx

```diff
@@ -0,0 +1,25 @@
+import { connect } from "react-redux";
+import _ from "underscore";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { getUpgradeUrl } from "metabase/selectors/settings";
+import { State } from "metabase-types/store";
+
+interface NudgeProps {
+  upgradeUrl: string;
+}
+
+const NudgeToPro = (props: NudgeProps) => {
+  return (
+    <div>
+      TODO: Nudge to <ExternalLink href={props.upgradeUrl}>Pro</ExternalLink>
+    </div>
+  );
+};
+
+const mapStateToProps = (state: State) => ({
```

Comment:
It would be better to use `useSelector` in the component, for example:
```typescript
import { useSelector } from "metabase/lib/redux"

const NudgeToPro = () => {
  const upgradeUrl = useSelector(getUpgradeUrl(state, { utm_media: "admin_people" }))

  return (
    <div>
      TODO: Nudge to <ExternalLink href={upgradeUrl}>Pro</ExternalLink>
    </div>
  );
};
```
That way we can get rid of the `compose` and `connect` higher order components, as well as the prop that the parent doesn't actually use which is confusing.